source/content/terminus/02-install.md

```diff
@@ -95,14 +95,33 @@ The commands below will:
 - Make the file executable
 - Add a symlink to your local `bin` directory for the Terminus executable
 
+<TabList>
+
+<Tab title="PHP 8.4+" id="php84" active={true}>
```

Comment:
8.2+

Reply:
This is confusing. We're breaking it down by when to install Terminus 3 vs Terminus 4. The reason you'd want to install Terminus 4 is because you are using PHP 8.4. I guess I should break that down in the titles instead.